### ISSUE TITLE
Twig syntax support

### ISSUE THREAD

**Issue by iHedgehog** [jonschlinkert/sublime-markdown-extended#88]
Please add twig syntax support.

**Comment by jonschlinkert** [comment]
What file extensions are typically associated with twig? I would guess `twig`? any abbreviations or others? I'll add it when I get a chance unless you or someone else would like to do a pr...?

**Comment by iHedgehog** [comment]
@jonschlinkert yes, `twig`: `filename.html.twig`, `filename.json.twig`, etc... Thanks.

**Comment by jonschlinkert** [comment]
noted, thanks for the info
